rclcpp/include/rclcpp/executable_list.hpp

```diff
@@ -38,7 +38,31 @@ struct ExecutableList
   ExecutableList();
 
   RCLCPP_PUBLIC
-  virtual ~ExecutableList();
+  ~ExecutableList();
```

Comment:
why removed `virtual` ?

Reply:
Why have it? We don't need polymorphism and this class is not intended to be inherited and modified